### PR TITLE
docs: remove demos

### PR DESCRIPTION
**Pull request by langz**
This PR removes the following pages(and related files) in our docs:
- https://eufemia.dnb.no/uilib/getting-started/demos/
- https://eufemia.dnb.no/uilib/demos/
- https://eufemia.dnb.no/uilib/demos/eufemia-demo/

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/usage/first-steps/react.mdx [hunk 1]
```diff
@@ -38,8 +38,6 @@ import { hamburger as hamburgerIcon } from '@dnb/eufemia/icons'
 
 Now, let's jump to the [Button Docs](/uilib/components/button) for more details.
 
-Also, you may be interested in more code examples? Have a look at the [Demo App on GitHub](https://github.com/dnbexperience/eufemia-demo/blob/main/src/form-demos/form-demo-01.js) or [Example for React](https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-react).
-
 ## Examples
 
-Have a look at [the demos](/uilib/getting-started/demos) as well as a [Webpack example app](https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-react) and [SSR example](https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-ssr).
+Have a look at the [Webpack example app](https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-react) and [SSR example](https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-ssr).
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/usage/first-steps/react.mdx:43]: Should I also remove the references/links to these two examples?
- https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-react
- https://github.com/dnbexperience/eufemia-examples/tree/main/packages/example-ssr

Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/usage/first-steps/react.mdx:43]: Yes, we an remove them as well.

**Review by tujoworker — APPROVED** [review]
And into the bin with it 🚀

**Pull request merged**